playbooks/webapp/tasks/create_aurora_db_cluster.yaml

```diff
@@ -0,0 +1,95 @@
+      security_token: "{{ security_token | default(omit) }}"
+  block:
+    - name: Get security group id - primary region
+      amazon.aws.ec2_security_group_info:
+        filters:
+          group-name: "{{ rds_secgroup_name }}"
+        region: "{{ test_primary_cluster_region }}"
+      register: rds_primary_sg
+
+    - name: Get security group id - replica region
+      amazon.aws.ec2_security_group_info:
+        filters:
+          group-name: "{{ rds_secgroup_name }}"
+        region: "{{ test_replica_cluster_region }}"
+      register: rds_replica_sg
+
+    - name: Create  Aurora db cluster
+      ansible.builtin.include_role:
+        name: cloud.aws_ops.create_rds_global_cluster
+      vars:
+        create_rds_global_cluster_operation: create
+        create_rds_global_cluster_engine: "aurora-postgresql"
+        create_rds_global_cluster_engine_version: "{{ rds_engine_version }}"
+        create_rds_global_cluster_instance_class: "{{ test_instance_class }}"
+        create_rds_global_cluster_master_username: "{{ deploy_flask_app_rds_master_username }}"
+        create_rds_global_cluster_master_user_password: "{{ deploy_flask_app_rds_master_password }}"
+        create_rds_global_cluster_global_cluster_name: "{{ test_global_cluster_name }}"
+        create_rds_global_cluster_primary_cluster_name: "{{ test_primary_cluster_name }}"
+        create_rds_global_cluster_primary_cluster_region: "{{ test_primary_cluster_region }}"
+        create_rds_global_cluster_primary_cluster_instance_name: "{{ test_primary_cluster_instance_name }}"
+        create_rds_global_cluster_replica_cluster_name: "{{ test_replica_cluster_name }}"
+        create_rds_global_cluster_replica_cluster_region: "{{ test_replica_cluster_region }}"
+        create_rds_global_cluster_replica_cluster_instance_name: "{{ test_replica_cluster_instance_name }}"
+        create_rds_global_cluster_db_subnet_group_name: "{{ rds_subnet_group_name }}"
+        create_rds_global_cluster_primary_cluster_db_name: "{{ rds_instance_name }}"
+        create_rds_global_cluster_primary_cluster_vpc_security_group_ids:
+          - "{{ rds_primary_sg.security_groups[0].group_id }}"
+        create_rds_global_cluster_replica_cluster_vpc_security_group_ids:
+          - "{{ rds_replica_sg.security_groups[0].group_id }}"
+
+    - name: Get primary instance info
+      amazon.aws.rds_instance_info:
+        db_instance_identifier: "{{ test_primary_cluster_instance_name }}"
+        region: "{{ test_primary_cluster_region }}"
+      register: primary_instance_info_result
+
+    - name: Get primary cluster info
+      amazon.aws.rds_cluster_info:
+        db_cluster_identifier: "{{ test_primary_cluster_name }}"
+        region: "{{ test_primary_cluster_region }}"
+      register: primary_cluster_info_result
+
+    - name: Get global db info
+      command: "aws rds describe-global-clusters --global-cluster-identifier {{  test_global_cluster_name }}"
```

Comment:
You should rather create an info module to describe the clusters otherwise this will add aws cli as requirements and this will also required for the user to either provide aws credentials as env variables or run `aws configure`

Reply:
I can take this up as a separate ticket and work on it based on priorities.

Reply:
https://github.com/ansible-collections/amazon.aws/issues/1804